### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by StephenWeatherford**
I won't merge before the current release unless we decide to combine them.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -63,7 +60,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<AzureE
         registerTableGroupActionHandlers();
 
         vscode.window.registerTreeDataProvider('azureStorage', tree);
-        registerCommand('azureStorage.refresh', async (treeItem?: AzureTreeItem) => await tree.refresh(treeItem));
+        registerCommand('azureStorage.refresh', async (treeItem?: AzureTreeItem) => tree.refresh(treeItem));
```
Comment (EricJizbaMSFT) [inline src/extension.ts]: nit: why the change to this?

Comment (StephenWeatherford) [reply src/extension.ts]: I think that was an auto-fix, I remember seeing lint complain about that but don't think I changed it.  Seems harmless here either way, you agree?

Comment (EricJizbaMSFT) [reply src/extension.ts]: Yeah I guess it probably doesn't make a difference
